operators/src/opencl/cl_program.rs

```diff
@@ -1006,17 +1150,56 @@ impl CompiledCLProgram {
                         // no geo
                     }
                     VectorDataType::MultiPoint => {
+                        kernel.arg_named(
+                            format!("IN_FEATURE{}_COORDS", idx),
+                            None::<&Buffer<Double2>>,
+                        );
+
+                        kernel.arg_named(format!("IN_FEATURE{}_LEN", idx), i32::zero());
```

Comment:
why not None::<i32>? then we know it it wasn't set properly

Reply:
Hm, but a None-pointer is of equivalent size as a int32 pointer, but i32 is 4 byte and None is... idk-bytes long?

Reply:
From the documentation

> Scalar and vector arguments may not be null; use zero (e.g. `&0`) instead.